sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_session.py

```diff
@@ -28,11 +28,11 @@
 
 
 class BaseSession(object):
-    def __init__(self, session_id, receiver, encoding="UTF-8"):
```

Comment:
would LOVE to change this to keyword only instead

Reply:
I think removing this is fine for me.
Session is something that user shouldn't create by themselves, however, at the moment it is exported due to documentation needs.

I'm still investigating whether we could stop exporting the class while not affecting the documentation, but just in the case we have to export it, I'd rather not having a kwargs which might confuse the user "what are in the kwargs they can set" and if we want we could add back later.

Reply:
I'm inclined not to because of what Adam said.  Kwarg hiding is workable for public surface areas, but still suboptimal there, and in a way that I'm nervous to expose us to internally: namely, you don't notice typod arguments any more.

Let me know if this isn't a convincing argument and I can try harder, don't want to come in and flip the table :P